README.md

```diff
@@ -13,8 +13,10 @@ Note : 2019 conferences are listed [here](year-pages/2019-conferences-list.md).
 | [FunctionalConf](https://functionalconf.com/) | 12-16 Dec | The LaLiT Ashok Hotel, Bengaluru | Bring together the growing community of functional programmers under one roof. |
 | [Devops Days India](http://devopsdaysindia.org/) | 8-9 Dec | Sheraton Grand, Bengaluru | Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. | 
 | [JSFoo](https://jsfoo.in/2018/) | 26-27 Oct | NIMHANS Convention Centre, Bengaluru | The eighth edition of India’s first JavaScript conference.|
+| [Meta Refresh](https://metarefresh.in/) | 26-27 Oct | NIMHANS Convention Centre, Dairy Circle, Bengaluru | On interaction design and CSS from product and business standpoints. |
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
 | [AWS Community Day Bengaluru](https://communityday.awsugblr.in/) | 05-06 Oct | The Ritz-Carlton, Bangalore | Use Cases / Success Stories and Workshops on Amazon Web Services |
+| [PySangamam](https://pysangamam.org/) | 07-08 Sept | IC & SR - IIT Madras | Tamil Nadu's first Python conference |
```

Comment:
nit: s/Sept/Sep

nit: Period/Full stop at the end of the sentence in the description.